### PR TITLE
Fix browser warnings

### LINKED ISSUE #7515: Address Browser Warnings
**Describe the bug**
‘PageRender’, ‘setupHome’, and ‘HomeListCard’ are missing a `key` prop in list rendering.

In ‘use_case_footer.tsx’
Could not find required 'intl' object. <IntlProvider> needs to exist in the component ancestry.

Navlink of workspace_list has already been registered in group settingsAndSetup

**To Reproduce**
Open the console in homepage to see the warnings.

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4944

### PR DESCRIPTION
**Pull request by Kapian1234**
### Description

Added `key` props in ‘PageRender’, ‘setupHome’, and ‘HomeListCard’.

Resolved 'intl' object not found error by using i18n.translate.

Resolved the issue of duplicate 'addNavLinksToGroup' calls to prevent the "Navlink of workspace_list has already been registered in group settingsAndSetup".

Modified the expect of addNavLinksToGroup in workspace plugin.test.js

### Issues Resolved

closes #7515

## Changelog

- fix:  Resolve some browser warnings

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by ruanyl — APPROVED** [review]
Nice work! Could you maybe remove the duplicate changelogs?

**Comment by Kapian1234** [comment]
> Nice work! Could you maybe remove the duplicate changelogs?

Ok. Thank you!

File: src/plugins/workspace/public/components/home_get_start_card/use_case_footer.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ import {
   EuiModalHeaderTitle,
 } from '@elastic/eui';
 import React, { useMemo, useState } from 'react';
-import { FormattedMessage } from 'react-intl';
+// import { FormattedMessage } from 'react-intl';
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/components/home_get_start_card/use_case_footer.tsx:25]:
```suggestion
```

File: src/plugins/workspace/public/components/home_get_start_card/use_case_footer.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ export const UseCaseFooter = ({
           onClick={showModal}
           data-test-subj="useCase.footer.createWorkspace.button"
         >
-          <FormattedMessage id="useCase.footer.createWorkspace" defaultMessage="Create workspace" />
+          {i18n.translate('useCase.footer.createWorkspace', { defaultMessage: 'Create workspace' })}
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/components/home_get_start_card/use_case_footer.tsx:97]:
```suggestion
          {i18n.translate('workspace.useCase.footer.createWorkspace', { defaultMessage: 'Create workspace' })}
```
The key should comply with the directory the file is in.

Comment (Kapian1234) [reply src/plugins/workspace/public/components/home_get_start_card/use_case_footer.tsx:97]: thx!

**Review by ruanyl — APPROVED** [review]

**Review by ruanyl — APPROVED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**